invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposits/RDMDepositForm/RDMDepositForm.js

```diff
@@ -376,20 +377,20 @@ export class RDMDepositForm extends Component {
                   labelIcon="tag"
                 />
 
-                {/* <LanguagesField
+                <LanguagesField
                   initialOptions={_get(
                     this.props.record,
                     "metadata.languages",
                     null
                   )}
                   serializeSuggestions={(suggestions) =>
                     suggestions.map((item) => ({
-                      text: item.metadata.title[this.config.current_locale],
+                      text: item.title_l10n,
                       value: item.id,
                       key: item.id,
                     }))
                   }
-                /> */}
+                />
                 <ComingSoonField
```

Comment:
Oups we should remove the coming soon for languages if languages is uncommented.

Reply:
Oups I am adding it in https://github.com/inveniosoftware/invenio-app-rdm/pull/496